PREDICTION_FOR_REPETITION

```diff
@@ -0,0 +1,66 @@
+package ArrayDataPrediction;
+
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Scanner;
+import java.util.Set;
+import GenerateData.DataGenerator;
+import GenerateData.Initiator;
+
+
+public class CheckForRepetetion {
+	public static void main(String[] args) {
+		int arraysize;	
+    
+		Scanner InputSize=new Scanner(System.in);
+		System.out.println("Enter the size of the array: ");
+		arraysize= InputSize.nextInt();
+		
+		DataGenerator dataGenerator = new DataGenerator(DataGenerator.PATTERN_REPEATED, arraysize);
+    System.out.println(dataGenerator.generate());
```

Comment:
Calling dataGenerator.generate() would create random array each time it is called. Therefore, the list you are printing here is not the same as the one that is generated in the next line. Hope you are aware of it.

Reply:
Yes! You are right. I missed that. 👍 
So, earlier the array was printed using the - "dataGenerator.generate()" function so it will always generate and print a different array as the previous line.
This will be solved by using "GetArray" list for printing, which is used to store the generated array.
Bingo! :)